qaframework-bdd-tests/src/test/resources/features/refapp-2.x/16-PhoneNumberValidator/phoneNumberValidator.feature

```diff
@@ -5,7 +5,7 @@ Feature:Phone Number validator
    
   @selenium
   @phoneNumberValidator
-  Scenario: Phone Number validator
+  Scenario: Phone Number validation
```

Comment:
isn't this change also supposed to be done elsewhere?

Reply:
Check my last commit

Reply:
checkout the `.yml` file name and its content.

Reply:
Resolved as needed.